src/components/link.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+import styles from './link.module.scss'
+
+export default class Link extends React.Component {
+  constructor(props) {
+    super(props)
+    this.text = props.children
```

Comment:
If these props change then the component won't correctly rerender since they props have been assigned to a field. Just use the values from props directly. If you'd like to make it a little easier to read you could do

```jsx
const { children, color, href } = this.props
```